src/top/data/colored_cube_dataset.py

```diff
@@ -11,18 +11,19 @@
 import torch.nn as nn
 import torch.nn.functional as F
 from torchvision.utils import save_image
-
-from pytorch3d.structures import Pointclouds
-from pytorch3d.transforms.transform3d import Transform3d
-from pytorch3d.renderer import (
-    look_at_view_transform,
-    FoVPerspectiveCameras,
-    PointsRasterizationSettings,
-    PointsRenderer,
-    PointsRasterizer,
-    AlphaCompositor,
-)
-
+try:
+    from pytorch3d.structures import Pointclouds
+    from pytorch3d.transforms.transform3d import Transform3d
+    from pytorch3d.renderer import (
+        look_at_view_transform,
+        FoVPerspectiveCameras,
+        PointsRasterizationSettings,
+        PointsRenderer,
+        PointsRasterizer,
+        AlphaCompositor,
+    )
+except ImportError:
+    print('welp')
```

Comment:
Ok, preferably let's fix this issue, but for now at least try to output helpful debug messages.
```suggestion
    logging.warn('pytorch3d import has failed, colored cube dataset will be disabled.')
```

Reply:
Thank you for suggestion.